app/models/comment.rb

```diff
@@ -67,6 +69,25 @@ class Comment < ApplicationRecord
 
   after_commit :remove_from_elasticsearch, on: [:destroy]
 
+  # [@atsmith813] this is adapted from the `search_field` property in
+  # `config/elasticsearch/mappings/feed_content.json` and
+  # `app/serializers/search/comment_serializer.rb`
+  #
+  # highlighter settings are taken from
+  # Search::QueryBuildersFeedContent#add_highlight_fields
+  pg_search_scope :search_comments,
+                  against: %i[body_markdown],
+                  using: {
+                    tsearch: {
+                      prefix: true,
+                      highlight: {
+                        StartSel: "<mark>",
+                        StopSel: "</mark>",
+                        MaxFragments: 2
```

Comment:
This is so slick, I absolutely love it. I don't feel like it should be a _model_ concern, to be clear, but it's pretty pragmatic considering Postgres is what's providing the highlighting.